cli/src/test/java/com/liferay/blade/cli/command/CreateCommandTest.java

```diff
@@ -1528,23 +1526,6 @@ public void testLiferayVersionDefault() throws Exception {
 		Assert.assertTrue(content, content.contains("\"com.liferay.portal.kernel\", version: \"4.4.0\""));
 	}
 
-	@Test
```

Comment:
Can you create a new command (hidden command) called `listProjectTemplates`, then you can readd this test.